File: 02.CSharpPartTwo/03_Methods/Methods/03EnglishDigit/EnglishDigit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*Problem 3. English digit

Write a method that returns the last digit of given integer as an English word.
 */

namespace _03EnglishDigit
{
    class EnglishDigit
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Enter number:");
            int number = int.Parse(Console.ReadLine());

            PrintLastDigit(number);
        }

        private static void PrintLastDigit(int number)
        {
            int lastDigit = number % 10;
            string[] digits = { "zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine" };

            Console.WriteLine(digits[lastDigit]);
        }
    }
}

File: 06_JavaScript/06_Functions/Functions/01.EnglishDigit/EnglishDigit.js
var number = 1024;

function returnLastDigit(inputNumber) {
    switch (inputNumber % 10) {
        case 0:
            console.log('zero');
            break;
        case 1:
            console.log('one');
            break;
        case 2:
            console.log('two');
            break;
        case 3:
            console.log('three');
            break;
        case 4:
            console.log('four');
            break;
        case 5:
            console.log('five');
            break;
        case 6:
            console.log('six');
            break;
        case 7:
            console.log('seven');
            break;
        case 8:
            console.log('eight');
            break;
        case 9:
            console.log('nine');
            break;
        default:
            console.log('Not a number...');
            break
    }
}

returnLastDigit(number);